### PR TITLE
update flax scheduler API

### PR DESCRIPTION
**Pull request by patil-suraj**
Align Flax scheduler API with PyTorch

### CODE REVIEW

File: src/diffusers/schedulers/scheduling_pndm_flax.py [hunk 1]
```diff
@@ -196,14 +199,31 @@ def set_timesteps(self, state: PNDMSchedulerState, num_inference_steps: int, sha
             )
 
         return state.replace(
-            timesteps=jnp.concatenate([state.prk_timesteps, state.plms_timesteps]).astype(jnp.int64),
```
Comment (patil-suraj) [inline src/diffusers/schedulers/scheduling_pndm_flax.py:199]: double precision values are not available by default injax cf https://github.com/google/jax#current-gotchas

**Comment by kashif** [comment]
LGTM!
